hermetic_build/README.md

```diff
@@ -25,13 +25,20 @@ for more information.
 
 ## Parameters to generate a repository using the docker image
 
-### Generation configuration yaml (`generation-config-path`)
+### Generation configuration yaml (`generation-config-path`), optional
 
 An absolute or relative path to a configuration file containing parameters to
 generate the repository.
 Please refer to [Configuration to generate a repository](#configuration-to-generate-a-repository)
 for more information.
 
+The default value is `$(pwd)/generation_config.yaml`, i.e., `generation_config.yaml`
+in the current working directory.
+
+Raise `FileNotFoundError` if the specified generation config does not exist or,
+in the case `generation-config-path` is not specified, the default generation
```

Comment:
```suggestion
This will raise `FileNotFoundError` if the specified generation config does not exist or,
in case `generation-config-path` is not specified, the default generation
```

Reply:
Done